fastcrypto-zkp/src/api.rs

```diff
@@ -44,6 +51,5 @@ pub fn verify_groth16_in_bytes(
         delta_g2_neg_pc_bytes,
     )?;
 
-    verify_with_processed_vk(&blst_pvk, &[x], &proof)
-        .map_err(|_| FastCryptoError::GeneralOpaqueError)
+    verify_with_processed_vk(&blst_pvk, &x, &proof).map_err(|_| FastCryptoError::GeneralOpaqueError)
```

Comment:
since there is nothing secret here, we should & can use a more explicit error (e.g., GeneralError(msg))

Reply:
That's fixed in #1623